x-pack/plugins/security_solution/cypress/tasks/create_new_rule.ts

```diff
@@ -449,6 +468,19 @@ export const fillEmailConnectorForm = (connector: EmailConnector = getEmailConne
   cy.get(EMAIL_CONNECTOR_PASSWORD_INPUT).type(connector.password);
 };
 
+export const fillIndexConnectorForm = (connector: IndexConnector = getIndexConnector()) => {
+  cy.get(CONNECTOR_NAME_INPUT).type(connector.name);
+  cy.get(COMBO_BOX_INPUT).type(connector.index.substring(0, 3));
```

Comment:
Why do we need `index.substring(0, 3)` here?

Locally, every time I run the test, it fails in this step:

<img width="312" alt="Screenshot 2022-09-27 at 17 04 24" src="https://user-images.githubusercontent.com/1938181/192563473-0df9d0e6-9b9a-4906-a681-33b672c6b0af.png">

Could that be because of the `substring(0, 3)`?

Reply:
That substring looks suspicious, indeed :)

Reply:
Actually that substring was intended.

@xcrzx the element you are sharing on the screenshot has been really problematic for me during the test development. I've faced the same issue as you are mentioning, when you are running Cypress on visual mode, if for whatever reason the focus of Cypress is lost because you are working in a different thing, that error appears. In order for that to don't happen you need when you start the test execution to don't change the focus to anything else.

About the substring, if the test typed the full index name, I was not able to find the proper element to click afterwards. More than happy to hear suggestions about doing it in a different way since I was not able to find a better solution :)

Reply:
I see no `substring` call! 🎉

Reply:
Not needed anymore now that we are not creating the index on the before hooks. Thanks again for the suggestion!! :)